src/spline_agent/dispatchers/http_dispatcher.py

```diff
@@ -0,0 +1,63 @@
+#  Copyright 2023 ABSA Group Limited
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import logging
+from urllib.parse import urljoin
+
+import requests
+from http_constants.headers import HttpHeaders
+from requests import Response
+
+from spline_agent.dispatcher import LineageDispatcher
+from spline_agent.json_serde import to_compact_json_str
+from spline_agent.lineage_model import ExecutionEvent, ExecutionPlan
+
+SPLINE_PRODUCER_PLANS_PATH = 'execution-plans'
+SPLINE_PRODUCER_EVENTS_PATH = 'execution-events'
+SPLINE_PRODUCER_CONTENT_TYPE_VER_12 = 'application/vnd.absa.spline.producer.v1.2+json'
```

Comment:
Should the '+' be here (in application/vnd.absa.spline.producer.v1.2+json)?

Reply:
yes, that's the standard format of a vendor specific MIME. The Backus-Naur form is the following:
```
mime-type = type "/" [tree "."] subtype ["+" suffix]* [";" parameter];
```
In Spline MIME above, the `vnd.` indicates the vendor tree, then it follows by the ABSA media subtype name and the version, and the suffix `+json` specifies the underlying MIME format. All together it means that the data format described by this MIME is JSON with the schema that is specific to _ABSA Spline Producer protocol_ of the version 1.2